requirements.txt

```diff
@@ -0,0 +1 @@
+deepdiff==3.3.0
```

Comment:
If a new top-level `test/` directory is created, can you move these requirements into it? To make it clear they're test-only.